src/test/scala/is/hail/annotations/UnsafeSuite.scala

```diff
@@ -0,0 +1,22 @@
+package is.hail.annotations
+
+import is.hail.SparkSuite
+import is.hail.check._
+import is.hail.check.Arbitrary._
+import is.hail.expr._
+import org.apache.spark.sql.Row
+import org.testng.annotations.Test
+
+class UnsafeSuite extends SparkSuite {
+
+  val g = for {
+    t <- Type.genStruct
+    a <- t.genNonmissingValue} yield (t, a)
```

Comment:
We should give much less size to the type than the value. At the least, something like the following will ensure their combined size is O(size).

```scala
def genStructTypeAndNonMissingValue: Gen[(Type, Annotation)] = for {
  (x, y) <- Gen.squareOfAreaAtMostSize
  t <- Type.genStruct.resize(x)
  v <- t.genNonMissingValue(y)
} yield (t, v)
```